visualstudio/2017/netstandard/NetstandardModuleSample/NetstandardModuleSample/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
+    "NetstandardModuleSample": {
+      "commandName": "Executable",
+      "executablePath": "$(OutDir)gw.exe",
+      "commandLineArgs": "module_dev_sample.json",
+      "environmentVariables": {
+        "Key": "Value"
+      }
+    }
+  }
+}
```

Comment:
This was a auto-generated file by VS2017 by editing the debug options in the solution properties. I can fix it, but I did not create the file and if someone else changes the setting the line-ending issue will be recreated by the IDE.